### ISSUE TITLE
debug dot file generator

### ISSUE THREAD

**Issue by woutersl** [cenotelie/hime#54]
**[Original report](https://bitbucket.org/cenotelie/hime/issue/54) by Anonymous.**

----------------------------------------

Hello.
There is an issue with the generated dot file. Specifically if terminals/productions have symbols like " or \.
The thing is viewer doesn't recognize labels like this:
  state114 -> state31 [label="\"];  -> error. Should be   state114 -> state31 [label="\\"];
  state0 -> state1 [label="""]; -> error. Should be state0 -> state1 [label="\""];

In short symbols like " or \ should be escaped in labels. Looks like this is trivial to fix.
